doc/releases.md

```diff
@@ -328,3 +328,19 @@ Close your release proposal PR and remove the proposal branch.
 ### 17. Celebrate
 
 _In whatever form you do this..._
+
+
+### 18. ARM6
+
+The ARM6 builds take significantly longer to build than other platforms
+and are more prone to failure. For this reason a release may be published
+in advance of the ARM6 build being ready.  
+
+The steps to follow when the ARM6 build must be promoted at a later time are:
+
+**a.** Validate that the build has completed.  If the build failed for some
+reason you can use the `iojs-release-arm6-only` build in the release CI to
+re-run the build only for ARM6.  When launching the build make sure to
+use the same commit hash as for the original release.
+
+**b.** Re-run steps 12-13 to promote and sign the release builds.
```

Comment:
It would be good to specify the names of those sections in case the numbering gets changed later.